docs/dxp/latest/en/site-building/site-navigation/using-navigation-menus.md

```diff
@@ -1,31 +1,29 @@
 # Using Navigation Menus
 
 Liferay provides the Navigation Menus application for creating custom menus and filters for Site Pages and content. With it, you can determine which elements are included and how they're organized. Once created, you can display your custom menus and filters using Page fragments and widgets.
```

Comment:
Hi @Fabiomorais87, could you please explain what filtering means in this context? Thx!

Reply:
Hi @jkappler, 

**Filtering** was to describe what you could choose to display, but after thinking about the text and talking with other TW I decided to take it out this term. 

This will be the new version: 

"Liferay provides the Navigation Menus application for creating custom menus for Site Pages and content. With it, you can determine which elements are included and how they're organized. Once created, you can display your custom menus using Page fragments and widgets."

Reply:
thanks for clarifying @Fabiomorais87!